src/components/SubjectViewer/SubjectViewer.js

```diff
@@ -24,10 +24,17 @@ const SubjectViewer = React.forwardRef(function ({
   setZoom,
   children,
 }, containerRef) {
-  if (!imageUrl || imageUrl.length === 0 || !containerRef) return null
+  useEffect(() => {
+    interactionRef.current && interactionRef.current.addEventListener('wheel', onWheel, { passive: false })
+    
+    return () => {
+      interactionRef.current && interactionRef.current.removeEventListener('wheel', onWheel)
+    }
+  })
```

Comment:
This runs on every prop and state change, adding and then removing an event listener. I'm not sure that's what it's intended to do.

Reply:
This might fix it. I'm not sure.
```suggestion
  useEffect(() => {
    interactionRef.current && interactionRef.current.addEventListener('wheel', onWheel, { passive: false })
    
    return () => {
      interactionRef.current && interactionRef.current.removeEventListener('wheel', onWheel)
    }
  }[interactionRef.current])
```

Reply:
This is a legit concern - the Event adding/removing is only supposed to occur on mount/unmount, not on every prop change. I've pushed a new commit that applies the changes you've suggested. 👌

Reply:
Cool. I'm not 100% sure when the returned cleanup function fires.